rs/nns/gtc/Cargo.toml

```
@@ -31,6 +30,7 @@ ic-nns-governance-api = { path = "../governance/api" }
 ic-nns-common = { path = "../common" }
 ic-nns-constants = { path = "../constants" }
 ic-nns-gtc-accounts = { path = "../gtc_accounts" }
+ic-secp256k1 = { path = "../../../packages/ic-secp256k1" }
```

Should this be
 
```suggestion
ic-secp256k1 = { workspace = true }
```

?

---

Thank you for the review! I would say in this case, since the `ic-secp256k1` path is always defined explicitly in every other package, I would leave it as is for consistency.